CONTRIBUTING.md

```diff
@@ -101,48 +101,57 @@ our lessons must run equally well on all three.
 
 ## Using GitHub
 
-If you choose to contribute via GitHub, you may want to look at
+If you choose to contribute via GitHub,
+you may want to look at
 [How to Contribute to an Open Source Project on GitHub][how-contribute].
-To manage changes, we follow [GitHub flow][github-flow].
-Each lesson has two maintainers who review issues and pull requests or encourage others to do so.
-The maintainers are community volunteers and have final say over what gets merged into the lesson.
-To use the web interface for contributing to a lesson:
-
-1.  Fork the originating repository to your GitHub profile.
-2.  Within your version of the forked repository, move to the `gh-pages` branch and
-create a new branch for each significant change being made.
-3.  Navigate to the file(s) you wish to change within the new branches and make revisions as required.
-4.  Commit all changed files within the appropriate branches.
-5.  Create individual pull requests from each of your changed branches
-to the `gh-pages` branch within the originating repository.
-6.  If you receive feedback, make changes using your issue-specific branches of the forked
-repository and the pull requests will update automatically.
-7.  Repeat as needed until all feedback has been addressed.
-
-When starting work, please make sure your clone of the originating `gh-pages` branch is up-to-date
-before creating your own revision-specific branch(es) from there.
-Additionally, please only work from your newly-created branch(es) and *not*
-your clone of the originating `gh-pages` branch.
-Lastly, published copies of all the lessons are available in the `gh-pages` branch of the originating
-repository for reference while revising.
+In brief:
+
+1.  The published copy of the lesson is in the `gh-pages` branch of the repository
+    (so that GitHub will regenerate it automatically).
+    Please create all branches from that,
```

Comment:
Replace with something like this (feel free to edit to make it clearer as needed):

1. The website for the lesson is generated from the `gh-pages` branch. The content in this branch is generated automatically by Travis CI from the `master` branch. Make your contributions by creating a new branch from `master`.  In this repository, please do *not* work from `gh-branch`.